infrastructure/src/test/java/org/corfudb/infrastructure/AbstractServerTest.java

```diff
@@ -0,0 +1,35 @@
+package org.corfudb.infrastructure;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import io.netty.channel.ChannelHandlerContext;
+import org.corfudb.protocols.wireprotocol.CorfuMsg;
+import org.corfudb.protocols.wireprotocol.CorfuMsgType;
+import org.junit.Test;
+
+public class AbstractServerTest {
```

Comment:
I don't understand what this class is suppose to do. The name AbstractServerTest would imply that it is being consumed by other tests.

Reply:
renamed to CorfuAbstractServerTest